deepspeed/runtime/engine.py

```diff
@@ -38,6 +40,8 @@
 from ..ops.adam import DeepSpeedCPUAdam
 from ..ops.adam import FusedAdam
 
+from deepspeed.profiling.flops_profiler.profiler import add_profile_methods, print_model_profile, print_model_aggregated_profile, flops_to_string, params_to_string
```

Comment:
Since we manage exports within each subpackage, we can just `from deepspeed.profiling import *` to expose the high-level API I think. That's a bit nicer than having a long list like this.

Since we do have this pattern of exporting a lot of methods, what about wrapping them up in a `FlopsProfiler` class?

Reply:
Done